src/daemon/control.rs

```diff
@@ -37,6 +37,7 @@ use revault_tx::{
     },
 };
 
+use jsonrpc_core::types::error::Error as JsonRpcError;
```

Comment:
i'm sorry if i induced you in error, we want `JsonRpcError`s in the `jsonrpc` module but it shoudl absolutely not leak in `control` (that's the point of separating it in the first place!).

Can we have more specialized enums here, pretty much what i tried to do in #196 with the `CommunicationError` and friends ?